libs/features/mas/src/global.css.js

```diff
@@ -147,6 +147,7 @@ merch-card.background-opacity-70 {
 }
 
 merch-card.has-divider hr {
+    margin-top: var(--consonant-merch-spacing-xs);
```

Comment:
Maybe we could join both lines into one:`margin: var(--consonant-merch-spacing-xs) 0;`?

Reply:
Right! Done.